pkg/icingaredis/heartbeat.go

```diff
@@ -78,26 +75,13 @@ func (h *Heartbeat) Err() error {
 	return h.err
 }
 
-// Lost returns a channel that will be closed if the heartbeat is lost.
-func (h *Heartbeat) Lost() <-chan struct{} {
-	return h.lost.Wait()
-}
-
-// Message returns the last heartbeat message.
-func (h *Heartbeat) Message() *v1.StatsMessage {
-	h.messageMu.Lock()
-	defer h.messageMu.Unlock()
-
-	return h.message
-}
-
 // controller loop.
 func (h *Heartbeat) controller(ctx context.Context) {
 	defer close(h.done)
 
 	h.logger.Info("Waiting for Icinga 2 heartbeat")
 
-	messages := make(chan v1.StatsMessage)
+	messages := make(chan *HeartbeatMessage)
```

Comment:
I don't care too much. Could probably even use the stats member to check for this. I don't think this will make a meaningful difference. This only happens once a second and the message contains a map (probably heap allocated) with value type `interface{}`, each probably also heap-allocated.

Reply:
So any further comment on this? I can change it but I doubt that it's worth the effort.

Reply:
Your decision.